src/js/shared/application-form.js

```diff
@@ -137,18 +136,18 @@ exports.createApplicationForm = function createApplicationForm(validateFile = tr
     }),
     team_apply: fields.boolean({
       label: 'Are you applying as part of a team?',
-      note: 'We will not process your application until you have been entered into a team via the application form (after submitting this form).',
-      widget: checkboxWidget('Yes, I am applying as part of a team. One of our members will fill out the team application form.'),
+      note: 'We won’t process your application until you’ve been entered into a team using the application form, after submitting this form.',
+      widget: checkboxWidget('Yes, I am applying as part of a team. One team member will fill out the team application form.'),
       cssClasses,
     }),
     team_placement: fields.boolean({
-      label: 'If not, would you like us to place you in a team?',
-      note: 'We can suggest a team for you before the event. You can always change this by contacting us.',
-      widget: checkboxWidget('Yes, please place me in a team!'),
+      label: 'Would you like us to put you in a team?',
```

Comment:
I think "If not" here is good as it makes it clear you can't ask for both options.